pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -54,7 +54,7 @@ def _reinit_optimizers_with_oss(self):
                 optim_class = type(optimizer)
                 zero_optimizer = OSS(params=optimizer.param_groups, optim=optim_class, **optimizer.defaults)
                 if _FAIRSCALE_OSS_FP16_BROADCAST_AVAILABLE:
-                    is_fp16 = self.lightning_module.trainer.precision == 16
+                    is_fp16 = self.lightning_module.trainer.precision == "mixed"
```

Comment:
```suggestion
                   precision = self.lightning_module.trainer.precision
                   is_fp16 = precision == "mixed" or precision == 16
```
just in case we enable true fp16 later